resources/site-packages/quasar/daemon.py

```diff
@@ -181,6 +181,16 @@ def start_quasard(**kwargs):
         kwargs["env"] = env
         kwargs["close_fds"] = True
 
+    wait_counter = 0
+    while xbmc.getCondVisibility('Window.IsVisible(10140)'):
+        log.debug('Addon Settings opened, waiting before start')
+        xbmc.sleep(1000)
+        wait_counter += 1
+
+        # Too much to wait, try to run it
+        if wait_counter > 60:
```

Comment:
60 seconds sounds too short to wait for the user to choose initial settings. You can remove the comment too, what this does is obvious enough.